Package.swift

```diff
@@ -53,17 +54,14 @@ var targets: [Target] = [
     ),
     .target(
         name: "TuistCoreTesting",
```

Comment:
Is there any reason why this and some others have been kept?

Reply:
They're used in `tuist-cloud`.